ow_plexil/src/plans/InitializeLander.plp

```diff
@@ -0,0 +1,25 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Reset pan and tilt angles to 0 and stow the arm (if not already there)
+
+#include "plan-interface.h"
+
+InitializeLander:
+{
+  InOut Boolean initialize;
```

Comment:
See my comments in calling plan about this variable.  This plan is simple and safe enough to context free and idempotent.

Reply:
I wanted to keep this initialization node the same for all the plans, which is why this initialize variable is included. It solves an issue when the initialization node is called at the same level as the mission node under a concurrence parent so that the mission doesn't start during initialization. Do you want a different initialization node for this first plan or can I keep this initialize variable so it can stay the same?